### ISSUE TITLE
Export reference in figure caption with org-ref-export

### ISSUE THREAD

**Issue by vnckppl** [jkitchin/org-ref#945]
I have an org file where I have a citation in a figure caption like so:

```
#+NAME: Figure1
#+CAPTION: Some text cite:&johnson-2010-great-paper some more text.
#+ATTR_ORG: :width 300px
#+ATTR_LATEX: :height 20cm
```

When I export this to a pdf file with org using `C-c C-e l p` it is being exported without any issues. However, when I export it with org-ref-export using `C-c C-e r p` I receive the error: `Citation johnson-2010-great-paper' on page 10 undefined on input line 238`. Not a real issue for me currently, because the default org convert is working, but maybe something worth looking into.

**Comment by jkitchin** [comment]
I can't reproduce that, can you put a more full example that shows this?

**Comment by vnckppl** [comment]
Not sure what is going on here. Just entering that text and converting it does not result in an error, but in my larger document it does. I have already removed all headers pointint to LaTeX pacakges except `#+LATEX_HEADER: \usepackage{svg}`. I got to a point where it exports to pdf, but then I still get the warning `LaTeX Warning: There were undefined references.` and there is a question mark where the citation in the caption should be. I will close this issue for now, because it seems to be on my end. I will comment when I have figured it out. Thanks for your prompt reply!
